setup.py

```diff
@@ -33,6 +33,16 @@
 
 libcloud.utils.misc.SHOW_DEPRECATION_WARNING = False
 
+# Different versions of python have different requirements.  We can't use
+# libcloud.utils.py3 here because it relies on backports dependency being
+# installed / available
+PY2 = sys.version_info.major == 2
```

Comment:
This won't work with Python version <= 2.6 where `sys.version_info` is a tuple. I will fix it when merging the patch.